dbt_sugar/core/task/doc.py

```diff
@@ -47,29 +50,15 @@ def run(self) -> int:
         schema = self._flags.schema
 
         dbt_credentials = self.load_dbt_credentials()
-        type_of_connection = dbt_credentials.get("type", "")
-
-        if type_of_connection == "postgres":
-            try:
-                columns_sql = PostgresConnector(
-                    user=dbt_credentials.get("user", str()),
-                    password=dbt_credentials.get("password", str()),
-                    host=dbt_credentials.get("host", str()),
-                    database=dbt_credentials.get("database", str()),
-                ).get_columns_from_table(model, schema)
-            except OperationalError as e:
-                raise RuntimeError(
-                    f"Could not connect to your postgres database, check your profiles.yml\n{e}"
-                )
-        elif type_of_connection == "snowflake":
-            columns_sql = SnowflakeConnector(
-                user=dbt_credentials.get("user", str()),
-                password=dbt_credentials.get("password", str()),
-                account=dbt_credentials.get("account", str()),
-                database=dbt_credentials.get("database", str()),
-            ).get_columns_from_table(model, schema)
+        connector = DB_CONNECTORS.get(dbt_credentials.get("type", ""))
```

Comment:
can we name this `connector_type` so that we don't think that it is a connector object by mistake?

Reply:
sure!

Reply:
I think that is better the name connector,
because we are creating a connector in that variable, not a type.